### PR TITLE
REG-3 | render list of objects

### PR DESCRIPTION
**Pull request by nazarrachunn**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 
 function App() {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/todos")
+      .then((response) => response.json())
+      .then((json) => {
+        setItems(json);
+      });
```
Comment (taras-bilozir-globacap) [inline src/App.tsx]: Should be:
`.then(setItems)`

File: src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React!
-        </a>
+        {items.map((item, index) => {
+          return <p key={index}>{JSON.stringify(item)}</p>;
```
Comment (taras-bilozir-globacap) [inline src/App.tsx]: Should be:
`<p key={item.id}>{item.title}</p>`

**Review by taras-bilozir-globacap — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (nazarrachunn) [reply src/App.tsx]: Done

File: src/App.tsx (hunk 1 above)
Comment (nazarrachunn) [reply src/App.tsx]: Done

**Review by taras-bilozir-globacap — APPROVED** [review]
Looks good.

**Pull request merged**